src/components/vm/vmDetailsPage.jsx

```diff
@@ -87,15 +87,29 @@ export const VmDetailsPage = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const vmNameHeadline = (
+        <h2 className="vm-name">{vm.name}</h2>
+    );
+
+    const vmTitleHeadline = (
+        <h2 className="vm-name">
+            {vm.title}
+            <span style={{ fontSize: "var(--pf-global--FontSize--xl)", paddingLeft: "0.83em" }}>
```

Comment:
This 0.83 feels like a bit magic number.. Can you just maybe wrap this in a flex container with some spacing? 

```<Flex spaceItems={{ default: 'spaceItemsSm' }}>```

Reply:
The intention was having the same space ob both sides and the space in the right side is , according to the chrome debugger, determined by `margin-block-end` of `h2`  wich is set to 0.83em. I could not find it as a css variable so I just hardcoded it. There are probably better ways but I'm not a React expert. The Flex spaceItems seems to put spaces on both sides.

Reply:
Additionally - I just realized that here we put the vm.name in parenthesis. Since most users don't use title, we defiintely want to have the name outside of parenthesis, when the user did not specify any title for the VM.